frontend/src/v5/ui/routes/dashboard/projects/containers/containersList/containerListItem/containerListItem.component.tsx

```diff
@@ -144,27 +143,19 @@ export const ContainerListItem = ({
 					{container.lastUpdated ? formatDate(container.lastUpdated) : ''}
 				</DashboardListItemText>
 				<DashboardListItemIcon>
-					<Tooltip
-						title={
-							container.isFavourite
-								? <FormattedMessage id="containers.list.item.favourite.removeTooltip" defaultMessage="Remove from favourites" />
-								: <FormattedMessage id="containers.list.item.favourite.addTooltip" defaultMessage="Add to favourites" />
-						}
-					>
-						<FavouriteCheckbox
-							checked={container.isFavourite}
-							selected={isSelected}
-							onClick={(event) => {
-								event.stopPropagation();
-							}}
-							onChange={(event) => {
-								onFavouriteChange(
-									container._id,
-									!!event.currentTarget.checked,
-								);
-							}}
-						/>
-					</Tooltip>
+					<FavouriteCheckbox
+						checked={container.isFavourite}
+						selected={isSelected}
+						onClick={(event) => {
```

Comment:
I guess we could also move the onclick stoppropagation bit inside favouritecheckbox?

Reply:
@sanmont3drepo Good idea. The onChange and checked could even be moved into the control if you supply the component with a federationOrContainer object ..?

Reply:
I think  this is good enough.

Reply:
Done